websites/A/Astral/presence.ts

```diff
@@ -23,15 +23,12 @@ presence.on("UpdateData", async () => {
     domainPicker = document.getElementsByClassName("ant-select-open")[0];
 
    if (embedEditor && embedEditor.parentElement.style.display !== "none") {
-       presenceData.details = "";
        presenceData.state = "Editing their embed";
```

Comment:
instead of only setting state, only set the details.
*some on all those other edits*